wasmbinding/test/custom_query_test.go

```diff
@@ -265,7 +265,7 @@ func TestQueryArithmeticTwapToNow(t *testing.T) {
 	twap, err := strconv.ParseFloat(resp.Twap, 32)
 	require.NoError(t, err)
 
-	require.Equal(t, 0.5, twap)
```

Comment:
Hrmm, this doesn't make sense to me

Reply:
Can we print the twap above it, and see if its an issue with float or something?

Reply:
`RESP TWAP: 0.500000416666666666`

Reply:
Any luck debugging why this is off?

Reply:
Not sure really. My best guess is that this PR changes the accumulator to use MS, whereas this query and the subsequent `GetArithmeticTwapToNow` uses `UnixMilli`. Does that sound like it could be it?